scripts/check_unit.sh

```diff
@@ -21,7 +21,39 @@ if [ -f profile.out ]; then
 fi
 }
 
-# Running aries-framework-go unit test
+# First argument is the exit code.
```

Comment:
Unfortunately I couldn't get https://github.com/ory/dockertest to work properly with the EDV image. No matter what I tried to do, it refused to map the port I set it to so that it could be reached from the running go tests. Not sure why this is, since I was able to get CouchDB's port mapped just fine.

So, I had to resort to starting them in the scripts. All this code basically ensures that:

1. The containers are removed after the test is finished
2. If the containers are already running, they are killed automatically
3. Unexpected errors aren't swallowed